Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -400,51 +570,123 @@ protected virtual void SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
+					else if (!tintColor.IsDefault)
+					{
+						if (_backgroundDrawable == null)
+							_backgroundDrawable = _bottomNavigationView.Background;
+						_bottomNavigationView.SetBackgroundColor(tintColor.ToAndroid());
+					}
 				}
 			}
 			else
 			{
-				if (tintColor.IsDefault && _backgroundDrawable != null)
-					_tabLayout.SetBackground(_backgroundDrawable);
-				else if (!tintColor.IsDefault)
+				if (_disposed || _tabLayout == null)
+					return;
+
+				Color tintColor = Element.BarBackgroundColor;
+
+				if (Forms.IsLollipopOrNewer)
 				{
-					if (_backgroundDrawable == null)
-						_backgroundDrawable = _tabLayout.Background;
-					_tabLayout.SetBackgroundColor(tintColor.ToAndroid());
+					if (tintColor.IsDefault)
+						_tabLayout.BackgroundTintMode = null;
+					else
+					{
+						_tabLayout.BackgroundTintMode = PorterDuff.Mode.Src;
+						_tabLayout.BackgroundTintList = ColorStateList.ValueOf(tintColor.ToAndroid());
+					}
+				}
+				else
+				{
+					if (tintColor.IsDefault && _backgroundDrawable != null)
+						_tabLayout.SetBackground(_backgroundDrawable);
+					else if (!tintColor.IsDefault)
+					{
+						if (_backgroundDrawable == null)
+							_backgroundDrawable = _tabLayout.Background;
+						_tabLayout.SetBackgroundColor(tintColor.ToAndroid());
+					}
 				}
 			}
 		}
 
 		void UpdateBarTextColor()
 		{
-			if (_disposed || _tabLayout == null)
-				return;
+			if (Element.OnThisPlatform().GetTabsPlacement() == TabsPlacement.Bottom)
+			{
+				if (_disposed || _relativeLayout == null || _bottomNavigationView == null)
+					return;
+
+				int currentColor = _bottomNavigationView.ItemTextColor.DefaultColor;
 
-			int currentColor = _tabLayout.TabTextColors.DefaultColor;
+        if (!_defaultColor.HasValue)
+					_defaultColor = currentColor;
 
-			if (!_defaultColor.HasValue)
-				_defaultColor = currentColor;
+				Color newTextColor = Element.BarTextColor;
+				int newTextColorArgb = newTextColor.ToAndroid().ToArgb();
+				var color = newTextColor.ToAndroid();
+				color.A = 128;
+				int newTextColorArgbFade = color.ToArgb();
 
-			Color newTextColor = Element.BarTextColor;
-			int newTextColorArgb = newTextColor.ToAndroid().ToArgb();
+				int[] sChecked = { global::Android.Resource.Attribute.StateChecked };
+				int[] sDefault = { };
 
-			if (!newTextColor.IsDefault && currentColor != newTextColorArgb)
-				_tabLayout.SetTabTextColors(newTextColorArgb, newTextColorArgb);
-			else if (newTextColor.IsDefault && _defaultColor.HasValue && currentColor != _defaultColor)
-				_tabLayout.SetTabTextColors(_defaultColor.Value, _defaultColor.Value);
+				if (!newTextColor.IsDefault && currentColor != newTextColorArgb)
+				{
+					_bottomNavigationView.ItemTextColor = new ColorStateList(new[] { sChecked, sDefault }, new[] { newTextColorArgb, color });
+					_bottomNavigationView.ItemIconTintList = new ColorStateList(new[] { sChecked, sDefault }, new[] { newTextColorArgb, color });
```

Comment:
Do we want the BarTextColor to also set the Tint?  Can we make the Tint Color a different settable property? Then we could also try applying said tint setting to the top bar as well